src/domainModules/collectionMammals/components/MammalForm/index.spec.js

```diff
@@ -243,16 +243,26 @@ describe('domainModules/collectionMammals/components/MammalForm', () => {
       },
       {
         id: 'add-feature-observation',
+        ignore: true,
         interaction: 'click',
       },
       {
+        ignore: true,
+        interaction: 'click',
+        name:
+          'featureObservations[1].featureObservationType.featureObservationTypeName',
+      },
+      {
+        interaction: 'click',
         name:
           'featureObservations[1].featureObservationType.featureObservationTypeName',
         selector: ({ form, name }) => {
-          const first = form.find({ name })
-          return first.find('input').hostNodes()
+          return form
```

Comment:
it's not clear from this code that it intends to select the `age` featureObservationTypeName. maybe passing a named function (could be inline) to the selector would solve that.